lib/apiservers/engine/backends/container.go

```diff
@@ -602,10 +602,12 @@ func (c *Container) ContainerInspect(name string, size bool, version version.Ver
 	// FIXME: Just set a bunch of dummy info needed to get docker attach working.
 	// Once we have event stream working (part of docker ps work), we need to go
 	// back and replace this.
+	running, _ := IsContainerRunning(vc.ContainerID)
```

Comment:
Two different API calls.  Sure, the CLI will call both inspect and attach during an attach, but the API doesn't know that.  We could reuse the value in the cache in ContainerAttach, but we can't know when inspect was last called.  We can't trust that value.  We need to check the portlayer again.